GLAA.ViewModels/LicenceApplication/EligibilityViewModel.cs

```diff
@@ -1,20 +1,83 @@
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Linq;
+using GLAA.ViewModels.Attributes;
 
 namespace GLAA.ViewModels.LicenceApplication
 {
     public class EligibilityViewModel : Validatable
     {
-        public SuppliesWorkersViewModel SuppliesWorkers { get; set; } = new SuppliesWorkersViewModel();
+        public EligibilityViewModel()
+        {
+            FullName = new PrincipalAuthorityFullNameViewModel();
+            EmailAddress = new PrincipalAuthorityEmailAddressViewModel();
+            Address = new AddressViewModel();
+            CommunicationPreference = new CommunicationPreferenceViewModel();
+            Password = new PasswordViewModel();
+        }
 
-        public OperatingIndustriesViewModel OperatingIndustries { get; set; } = new OperatingIndustriesViewModel();
+        public PrincipalAuthorityFullNameViewModel FullName { get; set; }
+        public PrincipalAuthorityEmailAddressViewModel EmailAddress { get; set; }
+        public AddressViewModel Address { get; set; }
+        public CommunicationPreferenceViewModel CommunicationPreference { get; set; }
+        public PasswordViewModel Password { get; set; }
+    }
 
-        public TurnoverViewModel Turnover { get; set; } = new TurnoverViewModel();
+    //TODO Replace FullNameViewModel with this
+    public class PrincipalAuthorityFullNameViewModel
+    {
+        [Display(Name = "Title")]
+        public string Title { get; set; }
 
-        public EligibilitySummaryViewModel EligibilitySummary { get; set; }
+        [Required]
+        [Display(Name = "First name")]
+        public string FirstName { get; set; }
 
+        [HiddenOptional]
+        [Display(Name = "Middle name")]
+        public string MiddleName { get; set; }
+
+        [Required]
+        [Display(Name = "Last name")]
+        public string LastName { get; set; }
+
+        [Display(Name = "Full name")]
+        public string FullName => (string.IsNullOrEmpty(Title) ? string.Empty : $"{Title} ") +
+                                  (string.IsNullOrEmpty(FirstName) ? string.Empty : $"{FirstName} ") +
+                                  (string.IsNullOrEmpty(MiddleName) ? string.Empty : $"{MiddleName} ") +
+                                  (string.IsNullOrEmpty(LastName) ? string.Empty : $"{LastName} ");
+    }
+
+    public class PrincipalAuthorityEmailAddressViewModel
+    {
+        [Required(ErrorMessage = "The Email Address field is required")]
+        [EmailAddress(ErrorMessage = "The Email Address field is not a valid e-mail address.")]
+        [Display(Name = "This will be the email address you use to log in to the system once it has been verified.")]
+        public string EmailAddress { get; set; }
+    }
+
+    public class PasswordViewModel : IValidatable
+    {
+        [Required]
+        [StringLength(100, ErrorMessage = "The {0} must be at least {2} characters long.", MinimumLength = 6)]
```

Comment:
I think Martin sent an email with some initial password requirements, might be worth incorporating that for this release. I think it was 8 chars, 1 upper 1 digit

Reply:
No problem